src/components/clusters/ClustersTable.tsx

```diff
@@ -115,7 +115,15 @@ const ClustersTable: React.FC<ClustersTableProps> = ({ rows, deleteCluster }) =>
 
   const rowFilter = React.useCallback(
     (row: IRow) => {
-      if (searchName && !(row.props.name || '').toLowerCase().includes(searchName.toLowerCase())) {
+      const searchableProps: string[] = [
+        row.props.name,
+        row.props.id,
+        row.props.baseDnsDomain,
+      ].map((prop) => (prop || '').toLowerCase());
+      if (
+        searchString &&
+        !searchableProps.filter((prop) => prop.includes(searchString.toLowerCase())).length
```

Comment:
nit: This can be simplified by using `Array.find()`

Reply:
done